README.md

```diff
@@ -17,3 +17,4 @@ Topics:
 First Name | Last Name | Education | Interests
 --- | --- | --- | ---
 Javad | Ebadi | **Physics, PhD** | Physics, Programming, Data Science
+Mohammadreza | Ebrahimi | **Physics, M. Sc** | High Energy Physics, Machine Learning, Data Science
```

Comment:
The **M. Sc** has extra space. The correct one is **M.Sc** or **MSc** or **MS**. Please replace that with one of these.